### DISCUSSION TITLE
How does vercel pro work

### DISCUSSION THREAD

**Discussion by ljrodriguez1** [vercel/vercel#6029] [Help]
I am considering using vercel for a project. It needs to be used on an organizarion repo, and because of that i need the pro plan.

Is the pro plan only needed for the user managing vercel, or should every member that commits to the repo have a pro plan.

If the answer is no, will a build be started if a user with no pro plan merges to master?

**Comment by mcsdevv** [comment]
Hi @ljrodriguez1, anyone wanting to make deployments for the project will need to be part of the Pro team, if not, the deployment will fail with an error stating this.
